src/ripple/app/misc/FeeEscalation.md

```diff
@@ -91,19 +91,34 @@ is opened up for normal transaction processing.
 4. A transaction in the queue can stay there indefinitely in principle,
 but in practice, either
   * it will eventually get applied to the ledger,
+  * it will attempt to apply to the ledger and fail,
+  * it will attempt to apply to the ledger and retry [10
+  times](#other-constants),
   * its last ledger sequence number will expire,
   * the user will replace it by submitting another transaction with the same
-  sequence number and a higher fee, or
+  sequence number and at least a 25% higher fee, or
   * it will get dropped when the queue fills up with more valuable transactions.
   The size limit is computed dynamically, and can hold transactions for
   the next [20 ledgers](#other-constants).
-  The lower the transaction's fee, the more likely that it will get dropped if the
-  network is busy.
 
-Currently, there is an additional restriction that the queue can only hold one
-transaction per account at a time. Future development will make the queue
-aware of transaction dependencies so that more than one can be
-queued up at a time per account.
+If a transaction is submitted for an account with one or more transactions
+already in the queue, and a sequence number that is sequential with the other
+transactions in the queue for that account, it must pay a
+[fee level](#fee-level) that is greater than 90% lower than the
+transaction with the previous sequence number, and will be considered
```

Comment:
Confusing. Say we have tx A with sequence 1, fee 100 and then tx B with sequence 2 from the same account. Does this mean that B must pay a fee greater than 10 or the amount of tx A (as mentioned) or something larger than 110% of the tx A fee? You might be able to improve the wording for clarity, same with multi_txn_percent.